sales_tax.rb

```diff
@@ -0,0 +1,152 @@
+=begin
+
+Basic sales tax is applicable at a rate of 10% on all goods, except books,
+food, and medical products that are exempt. Import duty is an additional
+sales tax applicable on all imported goods at a rate of 5%, with no
+exemptions.
+
+When I purchase items I receive a receipt which lists the name of all the
+items and their price (including tax), finishing with the total cost of the
+items, and the total amounts of sales taxes paid. The rounding rules for
+sales tax are that for a tax rate of n%, a shelf price of p contains
+(np/100 rounded up to the nearest 0.05) amount of sales tax.
+
+=end
+
+class Exempt
+
+  attr_accessor :quantity,:price,:name,:tax
+
+  def initialize( quantity , price , name)
+    @quantity = quantity
+    @price = price
+    @name = name
+    @tax = 1
+		
+  end
+	
+end
+
+class Taxed
+
+  attr_accessor :quantity,:price,:name,:tax
+
+  def initialize( quantity , price , name)
+    @quantity = quantity
+    @price = price
+    @name = name
+    @tax = 1.10
+  end
+
+end
+
+$final_output = []
+$no_tax = %w(chocolate chips snacks cake muffins pill tablet injection book)
+#add any keywords associated with tax exemption to $no_tax array
+$words = []
```

Comment:
Do we need global variables? Generally we don't go for global variables because in the large scale projects, global variables can be changed/accessed from everywhere so it becomes really hard to track the changes.